scripts/Engineering/EnggUtils.py

```diff
@@ -232,179 +93,39 @@ def plot_tof_vs_d_from_calibration(diag_ws, ws_foc, dspacing, calibration):
-                return ADS.retrieve("NorthBank_grouping")
-            grp_ws = mantid.LoadDetectorsGroupingFile(InputFile="ENGINX_North_grouping.xml",
-                                                      OutputWorkspace="NorthBank_grouping")
-            return grp_ws
-        except ValueError:
-            logger.notice("NorthBank grouping file not found in user directories - creating one")
-        bank_name = "NorthBank"
-    elif bank == 2:
-        try:
-            if ADS.doesExist("SouthBank_grouping"):
-                return ADS.retrieve("SouthBank_grouping")
-            grp_ws = mantid.LoadDetectorsGroupingFile(InputFile="ENGINX_South_grouping.xml",
-                                                      OutputWorkspace="SouthBank_grouping")
-            return grp_ws
-        except ValueError:
-            logger.notice("SouthBank grouping file not found in user directories - creating one")
-        bank_name = "SouthBank"
-    else:
-        raise ValueError("Invalid bank number given")
-    ws_name = bank_name + "_grouping"
-    grp_ws, _, _ = mantid.CreateGroupingWorkspace(InputWorkspace=sample_raw, GroupNames=bank_name,
-                                                  OutputWorkspace=ws_name)
-    return grp_ws
-
-
-def create_grouping_workspace_from_calfile(calfile, sample_raw):  # -> GroupingWorkspace
-    ws_name = "Custom_calfile_grouping"
-    grp_ws, _, _ = mantid.CreateGroupingWorkspace(InputWorkspace=sample_raw, OldCalFilename=calfile,
-                                                  OutputWorkspace=ws_name)
-    return grp_ws
-
-
-def create_grouping_workspace_from_spectra_list(str_list, sample_raw):
-    grp_ws, _, _ = mantid.CreateGroupingWorkspace(InputWorkspace=sample_raw, OutputWorkspace="Custom_spectra_grouping")
-    int_spectrum_numbers = create_spectrum_list_from_string(str_list)
-    for spec in int_spectrum_numbers:
-        ws_ind = int(spec - 1)
-        det_ids = grp_ws.getDetectorIDs(ws_ind)
-        grp_ws.setValue(det_ids[0], 1)
-    return grp_ws
-
-
 def default_ceria_expected_peaks(final=False):
     """
     Get the list of expected Ceria peaks, which can be a good default for the expected peaks
     properties of algorithms like PDCalibration
-
-    @param :: final - if true, returns a list better suited to a secondary fitting
+    @param :: final - if true, returns a list better suited to a secondary fitting of focused banks
     @Returns :: a list of peaks in d-spacing as a float list
     """
     _CERIA_EXPECTED_PEAKS = [1.104598643, 1.352851554, 1.631600313, 1.913220892, 2.705702376]
-    _CERIA_EXPECTED_PEAKS_FINAL = [0.855618487, 0.901900955, 0.914694494, 0.956610446, 1.04142562, 1.104598643,
-                                   1.210027059, 1.241461538, 1.352851554, 1.562138267, 1.631600313, 1.913220892,
-                                   2.705702376]
+    _CERIA_EXPECTED_PEAKS_FINAL = [0.781069, 0.855618487, 0.901900955, 0.914694494, 0.956610446, 1.04142562,
```

Comment:
Looks like there's an extra peak in here now? Similar to other question - what's the reason for this?

Reply:
This peak is relatively isolated considering it's at low d-spacing and seemed to be fit well consistently when I was testing so I added it. It should help get TZERO more accurately.